lambda.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os
+
+import boto3
+import django
+
+logger = logging.getLogger(__name__)
+
+
+def _set_env_variables(parameter_envs):
+    ssm = boto3.client("ssm")
+    response = ssm.get_parameters(Names=list(parameter_envs.keys()), WithDecryption=True)
+    logger.debug(f"SSM parameters: {response}")
```

Comment:
Will this debug statement reveal our secrets / passwords in plain text?

Reply:
Should be OK as the default logging level is `WARNING`: https://docs.python.org/3/howto/logging.html#when-to-use-logging (ie. debug statements aren't logged unless we change the logging level)